modules/AlphaWalletFoundation/AlphaWalletFoundation/Transactions/ApiNetworking.swift

```diff
@@ -36,3 +38,16 @@ public protocol ApiNetworking {
                                          pagination: TransactionsPagination,
                                          sortOrder: GetTransactions.SortOrder?) -> AnyPublisher<TransactionsResponse<TransactionInstance>, PromiseError>
 }
+
+extension ApiNetworking {
+    func log(response: URLRequest.Response) {
```

Comment:
Seems definition for `log(response:) is duplicated a few times. Can we extract them?